libcaf_net/caf/net/basp/worker.hpp

```diff
@@ -20,11 +20,11 @@
 
 #include <atomic>
 #include <cstdint>
-#include <vector>
 
 #include "caf/config.hpp"
 #include "caf/detail/abstract_worker.hpp"
 #include "caf/detail/worker_hub.hpp"
+#include "caf/fwd.hpp"
```

Comment:
A forward declaration is not enough in this file. The worker has a member of type `byte_buffer`, which requires the full type declaration.